app/components/feeds/FeedSelectorModal.module.css

```
@@ -0,0 +1,54 @@
+    
+    padding: var(--spacing16) 0 var(--spacing8);
+}
+
+.browseOpenFeedsButton {
+    font-family: var(--font-gentium-plus);
+    font-size: var(--text-lg);
+    font-weight: 400;
+    width: 100%;
+}
+
+@media(--tablet) {
```

Biome (2.3.14)

[error] Don't use unknown media feature names.

Unexpected unknown media feature name.
You should use media feature names defined in the CSS Specifications.

(lint/correctness/noUnknownMediaFeatureName)